.gitignore

```diff
@@ -19,7 +19,7 @@ tsconfig.tsbuildinfo
 /.env.development.local
 /.env.test.local
 /.env.production.local
-
+/.husky/post-checkout
```

Comment:
👋 @tunagonen! What’s the reasoning behind thus change in ignorefiles?

Reply:
hi alex, while trying merge and commit the code, i encountered the following prettier error which blocked me on finishing the merge. therefore added this as a fix:
![prettierError](https://user-images.githubusercontent.com/79975647/143490749-09311b73-184f-414b-9215-9a86fe42364b.png)

Reply:
Hmmm IC. Which terminal are you using? Try Git Bash or WSL on Windows. If this does not help, you can clear the contents of `.husky/post-checkout` locally but not stage it (i.e. not include it into commits).

It might be worth reverting whitespaces changes in ignorefiles to keep the diff clean.

Reply:
Actually it’s nothing to do with Windows and bash, it’s how our Prettier is configured. Try opening package.json and changing:

```diff
  "lint-staged": {
    "**/*": [
-     "prettier --write"
+     "prettier --write --ignore-unknown"
    ]
  },
```

Reply:
I’ll provide a more permanent fix later. Sorry for the inconvenience!